### ISSUE TITLE
Update/Audit docs 

### ISSUE THREAD

**Issue by sukritsingh** [mdtraj/mdtraj#1891]
Now that we have docs being built in CI, I think a next reasonable step is updating/auditing the docs as part of the next release. What do folks think? 

This thread will serve to track anything people find in the docs that might need updating.

**Comment by sukritsingh** [comment]
Brought up in #1772:

We should remove any mention of `unitcell_vectors` in the [Trajectory reference](https://mdtraj.readthedocs.io/en/latest/api/generated/mdtraj.Trajectory.html#mdtraj.Trajectory) - this is because `unitcell_vectors` is not an argument for `mdtraj.Trajectory.__init__` in the source code

**Comment by sukritsingh** [comment]
I've noticed that the current ReadTheDocs page takes _forever_ when doing a search. Example: https://mdtraj.readthedocs.io/en/latest/search.html?q=openmm&check_keywords=yes&area=default

Found something in [stack overflow](https://stackoverflow.com/questions/68709496/searching-takes-forever-on-readthedocs-when-the-phrase-is-not-present-on-th) that suggests that the fix here is updating our `sphinx_rtd_theme` 

We also probably need to update [`msmb_theme`](https://github.com/msmbuilder/msmb_theme) if we want to keep using it.

**Comment by mattwthompson** [comment]
I'd be fine using the off-the-shelf `sphinx_rtd_theme` if that theme needs a significant update; I'm a little surprised it seems to be used over there with no commits in 8 years https://github.com/search?q=org%3Amsmbuilder%20msmb_theme&type=code

**Comment by janash** [comment]
I agree that off-the-shelf `sphinx_rtd_theme` is likely the way to go as well. Are there any special features of the theme you've linked that would be missing?

**Comment by sukritsingh** [comment]
I think `msmb_theme` had some custom CSS that was consistent with MSMBuilder way back in the day - but I'm not sure what those exact differences are. I mostly kept `msmb_theme` since it was in all the previous versions of the docs and figured that kind of change would be a future PR. I'm not super attached to it if it doesn't change the docs significantly.
